ui/lib/pki/addon/components/page/pki-issuer-details.hbs

```diff
@@ -128,6 +128,20 @@
     {{/each-in}}
   {{/each}}
 </main>
+{{#if (or (eq @issuer.parsedCertificate.can_parse false) this.parsingErrors)}}
+  <AlertBanner
+    data-test-parsing-error-alert-banner
```

Comment:
Do we want to use the `HDS::Alert component` here since we started using it for tidy?

Reply:
I'm still wrapping up other PKI config work, so to save time I didn't think to update! But good question